cmake/scripts/linux/ArchSetup.cmake

```diff
@@ -9,6 +9,16 @@ set(PLATFORM_DIR platform/linux)
 set(CMAKE_SYSTEM_NAME Linux)
 if(WITH_ARCH)
   set(ARCH ${WITH_ARCH})
+elseif(CORE_PLATFORM_NAME_LC STREQUAL rbpi)
+  if(CPU STREQUAL arm1176jzf-s)
+    set(ARCH arm)
+    set(NEON False)
+    set(NEON_FLAGS "-mcpu=arm1176jzf-s -mtune=arm1176jzf-s -mfloat-abi=hard -mfpu=vfp")
+  elseif(CPU MATCHES "cortex-a7" OR CPU MATCHES "cortex-a53")
+    set(ARCH arm)
+    set(NEON True)
+    set(NEON_FLAGS "-fPIC -mcpu=${CPU} -mfloat-abi=hard -mfpu=neon-vfpv4 -mvectorize-with-neon-quad")
+  endif()
```

Comment:
this whole hunk is unnecessary, you just use -DENABLE_NEON=OFF and set you own cflags if you don't like those defaults

Reply:
IMO this piece of code is for cmake build

Reply:
Yes, but my statement is still valid. You can achieve the same with disabling neon or setting -DWITH_ARCH
 @lrusak removed Rbpi specific handling, so we probably shouldn't reintroduce it here.